ui/pages/token-details/token-details-screen.js

```diff
@@ -0,0 +1,159 @@
+import React, { useCallback, useContext } from 'react';
+import PropTypes from 'prop-types';
+import { I18nContext } from '../../contexts/i18n';
+import Tooltip from '../../components/ui/tooltip';
+import Button from '../../components/ui/button';
+import CopyIcon from '../../components/ui/icon/copy-icon.component';
+import { useCopyToClipboard } from '../../hooks/useCopyToClipboard';
+import Box from '../../components/ui/box';
+import Typography from '../../components/ui/typography';
+import {
+  COLORS,
+  TYPOGRAPHY,
+  FONT_WEIGHT,
+  DISPLAY,
+  TEXT_ALIGN,
+  OVERFLOW_WRAP,
+} from '../../helpers/constants/design-system';
+
+const TokenDetailsScreen = ({
+  address,
+  onClose = null,
+  onHideToken = null,
+  value,
+  icon,
+  currentCurrency,
+  decimals,
+  network,
+}) => {
+  const t = useContext(I18nContext);
+
+  const onHideTokenClick = useCallback(() => {
+    onHideToken();
+  }, [onHideToken]);
+
+  const [copied, handleCopy] = useCopyToClipboard();
+
+  return (
+    <Box className="page-container token-details">
+      <Box marginLeft={5} marginRight={6}>
+        <Typography
+          fontWeight={FONT_WEIGHT.BOLD}
+          margin={[4, 0, 0, 0]}
+          variant={TYPOGRAPHY.H6}
+          color={COLORS.BLACK}
+        >
+          {t('tokenDetailsTitle')}
+          <button onClick={onClose} className="token-details__closeButton" />
+        </Typography>
+        <Box display={DISPLAY.FLEX} marginTop={4}>
+          <Typography
+            align={TEXT_ALIGN.CENTER}
+            fontWeight={FONT_WEIGHT.BOLD}
+            margin={[0, 5, 0, 0]}
+            variant={TYPOGRAPHY.H4}
+            color={COLORS.BLACK}
+            className="token-details__token-value"
+          >
+            {value}
+          </Typography>
+          <Box marginTop={1}>{icon}</Box>
+        </Box>
+        <Typography
+          margin={[4, 0, 0, 0]}
+          variant={TYPOGRAPHY.H7}
+          color={COLORS.UI4}
+        >
+          {currentCurrency}
+        </Typography>
+        <Typography
+          margin={[6, 0, 0, 0]}
+          variant={TYPOGRAPHY.H9}
+          color={COLORS.UI4}
+          fontWeight={FONT_WEIGHT.BOLD}
+        >
+          {t('tokenContractAddress')}
+        </Typography>
+        <Box display={DISPLAY.FLEX}>
+          <Typography
+            variant={TYPOGRAPHY.H7}
+            margin={[2, 0, 0, 0]}
+            color={COLORS.BLACK}
+            overflowWrap={OVERFLOW_WRAP.BREAK_WORD}
+            className="token-details__token-address"
+          >
+            {address}
+          </Typography>
+          <Tooltip
+            position="bottom"
+            title={copied ? t('copiedExclamation') : t('copyToClipboard')}
+            containerClassName="token-details__copy-icon"
+          >
+            <button
```

Comment:
Should this be Button (capital B)?

Reply:
@segun I'm sorry I didn't answer this request before, I fix it here: https://github.com/MetaMask/metamask-extension/commit/fc353df083ca4099424224735ae80831198a6dcd